src/main/ts/ast/Remap.ts

```diff
@@ -35,8 +36,13 @@ const remapImport = (fs: FileSystem, remapCache: RemapCache, id: string, imp: Im
     fail(`Could not find export ${imp.fromName} in main module ${mainModulePath}`);
   }
 
+  // If the export is a variable declaration then we should't remap it
+  if (exportForImport.kind === ExportInfoKind.Variable) {
```

Comment:
Maybe I'm not reading this right but doesn't this mean that any Main.ts that has a var will no longer be remapped that means that the whole library of that thing will be imported if that variable is used. If this is the case we shouldn't do this automatically since that is the whole point of the remapped trying to avoid code bloat since we can't k ow when modules are pure. It's better that they are explicitly excluded since it's only the icon packs we need this for?

Reply:
Each export is a different node, so this will only apply if you do something like this:
```
export const a = 'a'`
```
and then in another file do
```
import { a } from 'something/otherfile';
```

Reply:
I also really don't like the idea of having a blacklist of things that just "fail" as we add more, it's likely just going to be more and more of a problem.

Reply:
I've just updated the `Remap` tests to show that the specified imports still work/remap when mixed with the variable declaration exports.

Reply:
I'm still not convinced that it solves anything if it ever includes the Main the whole file will be imported by rollup and all it's dependencies regardless if it's a constant or not. So all things must be remapped always except for type information but since swag runs after ts that is not a problem. I also still think we should require you to black list since otherwise you add one `let blah = something;` and boom you have code bloat in a release and we have no tools yet to see spikes in sizes so it would likely be released in that bloated state.

Reply:
I tested using the following updates to the oxide icon pack.

Added `one.js`:
```
const getOne = (name) => {
  console.log('not treeshaken');
}

export {
  getOne
}
```

and then in `icons.js` added the following:
```
import { getOne } from './one';

export var getAll = function () { return ({}); };

export {
  getOne
}
```

and all that was included in the output was the `getAll` function.

However if you still are worried, then I'll instead make this line a failure so that we exile all variable declaration exports in the main file and then add oxide to the blacklist.

Reply:
@spocke as discussed on slack, I've changed this to fail instead like it did previously, but with a different error message and added oxide to the ignore list. Let me know if you want anything else changed.